src/panels/OriginPanel/MenuButtons/CancelFlightButton.tsx

```diff
@@ -5,13 +5,14 @@ import {
   useGetStringPropertyValue,
   useOpenSpaceApi
 } from '@/api/hooks';
-import { AirplaneCancelIcon, AnchorIcon } from '@/icons/icons';
+import { AnchorIcon, CancelIcon } from '@/icons/icons';
 import { IconSize } from '@/types/enums';
-import { NavigationAnchorKey, ScenePrefixKey } from '@/util/keys';
+import { NavigationAnchorKey } from '@/util/keys';
+import { sgnUri } from '@/util/propertyTreeHelpers';
 
 export function CancelFlightButton() {
   const [anchor] = useGetStringPropertyValue(NavigationAnchorKey);
-  const anchorName = useGetPropertyOwner(`${ScenePrefixKey}${anchor}`)?.name ?? anchor;
+  const anchorName = useGetPropertyOwner(sgnUri(anchor))?.name ?? anchor;
```

Comment:
Use the GuiName property here?

Reply:
Hm, I don't know. I think I personally prefer to use the data we have stored about each property owner in redux, over accessing a property. Just makes more intutitive sense to me 🤔 otherwise, why store it?

Reply:
I think the first one is better